packages/tokamak/sdk/src/adapters/eth-bridge.ts

```diff
@@ -45,7 +44,7 @@ export class ETHBridgeAdapter extends StandardBridgeAdapter {
           from: event.args.from,
           to: event.args.to,
           l1Token: ethers.constants.AddressZero,
-          l2Token: predeploys.OVM_ETH,
+          l2Token: predeploys.ETH,
```

Comment:
I think we have to query NativeToken deposits for this.
l1Token: L1StandardBridge.nativeTokenAddress()
l2Token: predeploys.LegacyERC20NativeToken